docs/docs/cmd/entra/group/group-set.mdx

```diff
@@ -0,0 +1,99 @@
+import Global from '/docs/cmd/_global.mdx';
+
+# entra group set
+
+Updates a Microsoft Entra group
+
+## Usage
+
+```sh
+m365 entra group set [options]
+```
+
+## Options
+
+```md definition-list
+`-i, --id [id]`	
+: The ID of the Microsoft Entra group to update. Specify either `id` or `displayName` but not both.
+
+`-n, --displayName [displayName]`	
+: The display name of the Microsoft Entra group to update. Specify either `id` or `displayName` but not both.
+
+`--newDisplayName [newDisplayName]`	
+: The new display name of the Microsoft Entra group. The maximum length is 256 characters.
+
+`--description [description]`
+: The new description for the group.
+
+`--mailNickname [mailNickname]`
+: The new mail alias for the group (part before the @). Use only for mail-enabled groups. Maximum length is 64 characters.
+
+`--ownerIds [ownerIds]`
+: Comma-separated list of IDs of Microsoft Entra users that will be the group owners. Specify either `ownerIds` or `ownerUserNames`, but not both.
+
+`--ownerUserNames [ownerUserNames]`
+: Comma-separated list of UPNs of Microsoft Entra users that will be the group owners. Specify either `ownerIds` or `ownerUserNames`, but not both.
+
+`--memberIds [memberIds]`
+: Comma-separated list of IDs of Microsoft Entra users that will be the group members. Specify either `memberIds` or `memberUserNames`, but not both.
+
+`--memberUserNames [memberUserNames]`
+: Comma-separated list of UPNs of Microsoft Entra users that will be the group members. Specify either `memberIds` or `memberUserNames`, but not both.
+
+`--visibility [visibility]`
+: Specifies the group join policy and group content visibility for Microsoft 365 groups. Possible values are: `Private` or `Public`. Specify only when targeting a Microsoft 365 group.
+```
+
+<Global />
+
+## Remarks
+
+:::info
+
+The `visibility` option affects the behavior of the group.
+
+With the `Public` visibility:
+- Anyone can join the group without needing owner approval. 
+- Anyone can view the attributes of the group. 
+- Anyone can see the members of the group.
+
+With the `Private` visibilty:
+- Owner approval is needed to join the group.
+- Anyone can view the attributes of the group.
+- Anyone can see the members of the group.
+
+:::
+
+:::note
+
+This command allows using unknown options. For a comprehensive list of group properties, please refer to the [Graph documentation page](https://learn.microsoft.com/graph/api/resources/group?view=graph-rest-1.0#properties).
+
+:::
+
+If the specified option is not found, you will receive a `Resource 'xyz' does not exist or one of its queried reference-property objects are not present.` error.
+
+If the existing members or owners are not specified in the `memberIds`, `memberUserNames`, `ownerIds`, or `ownerUserNames` options, they will be removed from the group.
```

Comment:
Right now, people might think if they don't specify any of these options, we're removing everyone from the group. Let's try to rephrase it a bit, maybe something like:

Specifying `memberIds` or `memberUserNames` will make only those users members, removing all others. Similarly, specifying `ownerIds` or `ownerUserNames` will make only those users owners, removing all others.